### PR TITLE
Add supported OS list to readme

### PR DESCRIPTION
**Pull request by vjeffrey**
https://github.com/chef/inspec/issues/842

Let me know if that's ok/if more/different things are needed! :)
@chris-rock

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,6 +208,37 @@ Which will provide you with:
 {"family":"ubuntu","release":"14.04","arch":null}
 ```
 
+## Supported OS
+
+Remote Targets
+
+Platform | Versions | Architectures | Package Format
```
Comment (chris-rock) [inline README.md]: I think we do not need the package format column for remote targets.

Comment (vjeffrey) [inline README.md]: yup. that makes sense! just pushed up the fix

**Comment by chris-rock** [comment]
Forgot that, but we should add a warning for windows 2008 and windows 2008 R2 e.g. via a `*`. For those we need a specific version of powershell https://github.com/chef/inspec/issues/633

File: README.md [hunk 2]
```diff
@@ -208,6 +208,37 @@ Which will provide you with:
...
+OmniOS | | x86_64
+Gentoo Linux | | x86_64
+Arch Linux | | x86_64
+
+In addition, runtime support is provided for:
+  * Debian 8
+  * RHEL 6 & 7
+  * Ubuntu 12.04+
+  * Windows 7+
+  * Windows 2012+
```
Comment (arlimus) [inline README.md]: Could we use the same table style as above?

**Comment by arlimus** [comment]
Awesome job @vjeffrey ! would love to have the table style on the list below

**Comment by vjeffrey** [comment]
@arlimus @chris-rock updated :)

File: README.md [hunk 3]
```diff
@@ -208,6 +208,42 @@ Which will provide you with:
...
+Ubuntu Linux | | x86, x86_64
+SUSE Linux Enterprise Server  | 11, 12 | x86_64
+Scientific Linux | 5.x, 6.x and 7.x | i386, x86_64
+Fedora  | | x86_64
+OpenSUSE | 13.1/13.2/42.1 | x86_64
+OmniOS | | x86_64
+Gentoo Linux | | x86_64
+Arch Linux | | x86_64
+
+* Powershell with Windows Management Framework 5.0 required
```
Comment (chris-rock) [inline README.md]: Maybe: `For Windows 2008 and 2008 R2 an updated Powershell (Windows Management Framework 5.0) is required.`

**Comment by chris-rock** [comment]
Looks great. Tables are much better. Lets just make the Windows 2008 requirements a little more clear and we are good to go.

**Comment by vjeffrey** [comment]
should probably make sure this info eventually gets to: https://docs.chef.io/platforms.html

**Comment by chris-rock** [comment]
@vjeffrey this is only a restriction for inspec

**Comment by vjeffrey** [comment]
ya, I meant more like, is inspec gonna have a section on that page, since automate and a bunch of things are listed on there, as a one place to look at chef-things requirements?  just curious. also, why is the build failing?  that error msg is pretty vague..i assume it's just a transient error thing?

**Pull request merged**